vue/sbc-common-components/src/services/token.services.ts

```diff
@@ -111,21 +112,21 @@ class TokenServices {
   private syncSessionStorage () {
     if (this.kc) {
       if (this.kc.token) {
-        sessionStorage.setItem('KEYCLOAK_TOKEN', this.kc.token)
+        ConfigHelper.addToSession(SessionStorageKeys.KeyCloakToken, this.kc.token)
       }
       if (this.kc.refreshToken) {
-        sessionStorage.setItem('KEYCLOAK_REFRESH_TOKEN', this.kc.refreshToken)
+        ConfigHelper.addToSession(SessionStorageKeys.KeyCloakRefreshToken, this.kc.refreshToken)
       }
       if (this.kc.idToken) {
-        sessionStorage.setItem('KEYCLOAK_ID_TOKEN', this.kc.idToken)
+        ConfigHelper.addToSession(SessionStorageKeys.KeyCloakIdToken, this.kc.idToken)
       }
     }
   }
 
   decodeToken () {
     try {
-      let token = sessionStorage.getItem('KEYCLOAK_TOKEN')
-      if (token != null) {
+      let token = ConfigHelper.getFromSession(SessionStorageKeys.KeyCloakToken)
```

Comment:
Use `const` when property isn't reassigned later. This will avoid some code debt for when `strict=true` is set (or required by apps that use this).

Reply:
Changed.  BTW this has nothing to do with strict mode.